### ISSUE TITLE
wikiDetoxAnnotated40kRows.tsv unreliable?

### ISSUE THREAD

**Issue by rubenev** [dotnet/machinelearning-samples#346]
Not sure where to post this. but the wikiDetox dataset used in the tutorial for Sentiment Analysis seems unreliable. 22k neutral vs 2k negative comments gives a very low recall. Having an function that would just returns false for every testing data would have a precision of 90%.

**Comment by justinormont** [comment]
The label skew is expected, and a part of machine learning datasets in the wild. 

When using models for any dataset, look for AUC scores >0.50, and Log-loss reduction scores >0.0. These indicate your model is doing better than just picking the largest class and is true for all datasets. If you know the label skew of your particular dataset, you can also see that your other metrics are improving; so for WikiDetox, look for an accuracy >0.90. 

Generally, for WikiDetox, you'll see around 0.97 for the AUC, and 0.95 for the accuracy. 

For certain datasets accuracy isn't great: 
* very small datasets
* highly imbalanced (>20:1)
* noisy datasets
* very hard/easy (scores near 1.0/0.0)

**Comment by arafattehsin** [comment]
@rubenev I agree with you. I think it needs to be improved. The previous data set was much better.
